client/my-sites/backup/backup-contents-page/file-browser/file-browser-header.tsx

```diff
@@ -0,0 +1,46 @@
+import { Button } from '@wordpress/components';
+import { FunctionComponent } from '@wordpress/element';
+import { close } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+
+interface FileBrowserHeaderProps {
+	setShowCheckboxes: ( enabled: boolean ) => void;
+	showCheckboxes: boolean;
+}
+
+const FileBrowserHeader: FunctionComponent< FileBrowserHeaderProps > = ( {
+	setShowCheckboxes,
+	showCheckboxes,
+} ) => {
+	const translate = useTranslate();
+	const onSelectClick = () => {
+		setShowCheckboxes( true );
+	};
+	const onCancelClick = () => {
+		setShowCheckboxes( false );
+	};
+
+	return (
+		<div className="file-browser-header">
+			{ ! showCheckboxes && (
+				<Button
+					className="file-browser-header__select-button"
+					onClick={ onSelectClick }
+					isSecondary
```

Comment:
```suggestion
					variant="secondary"
```

Button component no longer support `isSecondary`, now we should use `variant="secondary"`.

Reference: https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/button/README.md#variant-primary--secondary--tertiary--link